### ISSUE TITLE
How to enable debug logging?

### ISSUE THREAD

**Issue by ilakast** [ebruck/radiotray-ng#114]
Built and installed my own i386 package from source following the instructions. When I run from the newly created menu, I cannot see any icon on the tray. The log shows:

`[2019-06-09 15:15:41.322897] [0xb4a56b80] [info ] (main.cpp:246) - radiotray-ng (v0.2.5-unknown) starting up
[2019-06-09 15:15:41.323159] [0xb4a56b80] [info ] (main.cpp:248) - radiotray-ng/0.2.5 (Linux i686; Ubuntu/18.10 (cosmic))
[2019-06-09 15:15:41.323214] [0xb4a56b80] [debug] (config.cpp:29) - loading: /home/kiosk/.config/radiotray-ng/radiotray-ng.json
[2019-06-09 15:15:41.323355] [0xb4a56b80] [info ] (main.cpp:90) - **debug logging disabled**
[2019-06-09 15:15:41.323408] [0xb4a56b80] [info ] (player.cpp:31) - starting gstreamer
`
Distributor ID:	Ubuntu
Description:	Ubuntu 18.10
Release:	18.10
Codename:	cosmic

Desktop is LXQt
`ls /usr/bin/*session`
gives
`/usr/bin/dbus-run-session  /usr/bin/lxqt-config-session  /usr/bin/lxqt-session  /usr/bin/openbox-session`

**Comment by ebruck** [comment]
You may need to run these post install commands:

https://github.com/ebruck/radiotray-ng/blob/master/debian/postinst

**Comment by ilakast** [comment]
Thank you but no difference.
Is there a command line flag to enable debug logging?

**Comment by ebruck** [comment]
It's a config setting and described in the readme.

**Comment by ilakast** [comment]
Thanks again; apologies for missing the config settings. You pointed me to the right direction and I managed to find a workaround.
Oddly, when I run radiotray-ng from the menu, instead of the radiotray icon, I get a "cog" icon; this is a laptop with a pen-enabled screen and I use xvkbd for text entry which has that cog as an icon.
But the config settings (although I could not enable the debug log despite changing to 'true', I manually set the "last-station" & "last-station-group" values to specific values. Then, running radiotray-ng --play works and I get the radiotray icon fine (and not the cog). Thank you very much once again for the work and help
